src/client.rs

```diff
@@ -485,32 +558,20 @@ mod tests {
 
     #[tokio::test]
     async fn test_get_user_tracks() {
-        let tracks = client().user(8553751).tracks().await.unwrap().unwrap();
```

Comment:
Can you re-add the 3 tests you've removed?

Reply:
I replaced `test_get_collections` with `test_paginate_user_tracks` and added `test_get_first_page_user_tracks`. I also removed `test_get_user_playlists` since it's the same logic with the new API. I'll add it back if having the coverage is important you. I also removed `test_get_user_tracks_from_permalink` because that code is already covered with `test_get_user_permalink` and `test_paginate_user_tracks.`

Reply:
I would actually prefer having `test_get_user_playlists` as the logic might be the same but there can still be issues with deserializing or the url we're accessing.